guides/updates/administration/vue3.md

```diff
@@ -0,0 +1,129 @@
+
+## FAQ
+
+Let's start with some frequently asked questions. These will also help you figure out if this update affects you.
+
+### Which extensions are affected by the Vue 3 update?
+App-based extensions aren't affected by these changes. However, if your extension is plugin-based and contains custom administration code, you likely need to do some refactoring.
+
+### Are there any breaking changes I should be aware of?
+Yes, Vue 3 introduced breaking changes. It's crucial to review the migration guide provided by Vue.js and this document for detailed information.
+
+### What steps should I follow to update my Shopware plugin to Vue 3?
+Typically, the process involves updating your project dependencies and modifying your code to adhere to Vue 3's API changes. Consult the Vue 3 documentation and this document's step-by-step instructions.
+
+### Can one plugin version be compatible with Shopware 6.5 and 6.6?
+Your plugin requires a new version in the Store. For instance, version `1.x` is for Shopware `6.5.x`, while version `2.0` is compatible with Shopware `6.6` and newer.
+
+### How can I check if my Shopware extension is compatible with Vue 3?
+You can verify compatibility by reviewing the extension's functionality and updating test suites according to this document.
+
+### Do I need to rewrite my extension to update to Vue 3?
+While some changes are required, a complete rewrite is not necessary. The amount of effort is dictated by your use of Vue's internal API.
+
+### Are tools or libraries available to facilitate the migration to Vue 3?
+Yes, there are tools and migration helpers that can automate certain aspects of the update process. You could start by enabling the Vue 3 rule set of `eslint`.
+
+### Where can I find support and community discussions about updating Shopware plugin to Vue 3?
+You can participate in discussions and seek help on the Shopware community Slack. There is a dedicated channel for this topic called `#vue3-update`.
+
+## External resources
+
+Here is a handpicked selection of external resources. This list provides a handy reference, granting you access to all the essential materials you might need.
+
+- (Vue 3 migration guide)[https://v3-migration.vuejs.org]
+- (Vue 3 breaking changes)[https://v3-migration.vuejs.org/breaking-changes/]
+- (Vue router migration guide)[https://router.vuejs.org/guide/migration/]
+- (Vue test utils migration guide)[https://test-utils.vuejs.org/migration/]
+
+## Step-by-step guide
+::: warning
+Depending on when you read this document, you still need to enable the `VUE3` feature flag.
+To be safe, enable the `VUE3` feature flag and restart your watcher.
```

Comment:
Maybe add a link to the feature flag documentation?

Reply:
None of the feature flag documentation explains how to activate them just how to add one :see_no_evil: